README.md

```diff
@@ -27,7 +28,7 @@ Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
-[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[x] Selects all ul's of class dropdown, as well as the children elements that are li's
```

Comment:
Close, but it's actually the first option. e.g. `div > p` will select all paragraphs which are immediate children of a div, while a space (`div p`) will select paragraphs in a div, regardless of whether they are immediate children, or grandchildren, etc.

The option you chose would be expressed using a comma: `ul.dropdown, ul.dropdown li`